cpp/epidemiology/abm/household.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+* Copyright (C) 2020-2021 German Aerospace Center (DLR-SC)
+*
+* Authors: Daniel Abele, Sascha Korf
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+
+#include "household.h"
+#include "epidemiology/utils/eigen.h"
+#include <string>
+
+
+namespace epi {
+
+/**
+ * Function that adds a number of the same members to a household.
+ * @param household_member A household member from the HouseholdMember class.
+ * @param number_of_members The amount of members to be added.
+ */
+void Household::add_members(HouseholdMember household_member, int number_of_members)
+{
+    for (auto i = 0; i < number_of_members; i++) {
+        m_household_members.push_back(household_member);
+    }
+    m_number_of_members += number_of_members;
```

Comment:
why don't you treat this households the same as householdgroups, i.e. store pairs of members and counts?

Otherwise, if you do it like this, why do you need to store the number of members at all, it's just the size of the `m_household_members` vector.

Reply:
Changed add_members to be rated the same as add_households